src/examples/java/com/nylas/examples/other/EventsExample.java

```diff
@@ -102,6 +93,16 @@ protected static void basicEventCrud(Events events, Calendar primary) throws IOE
 		metadata.put("event_category", "gathering");
 		created.setMetadata(metadata);
 
+		EventConferencing conferencing = new EventConferencing();
+		conferencing.setProvider("Zoom Meeting");
+		EventConferencing.Details details = new EventConferencing.Details();
+		details.setMeetingCode("213");
+		details.setPassword("xyz");
+		details.setUrl("https://us02web.zoom.us/j/****************");
+		details.setPhone(Collections.singletonList("+11234567890"));
+		conferencing.setDetails(details);
+		event.setConferencing(conferencing);
```

Comment:
this should be on the `created` variable, rather than the `event` object.

Reply:
Good catch, copy/paste error on my part lol